crates/forge-runner/src/test_case_summary.rs

```diff
@@ -216,11 +217,13 @@ impl TestCaseSummary<Single> {
         gas: u128,
         used_resources: UsedResources,
         call_trace: &Rc<RefCell<InternalCallTrace>>,
+        encountered_errors: &[EncounteredError],
         contracts_data: &ContractsData,
         maybe_versioned_program_path: &Option<VersionedProgramPath>,
     ) -> Self {
         let name = test_case.name.clone();
-        let msg = extract_result_data(&run_result, &test_case.config.expected_result);
+        let msg = extract_result_data(&run_result, &test_case.config.expected_result)
+            .map(|msg| add_backtrace_footer(msg, contracts_data, encountered_errors));
```

Comment:
Why are we calling `add_backtrace_footer` both in forge and forge-runner?

Reply:
Okay, never mind, it's both in forge-runner and I didn't realize it was two different branches.